Xamarin.Forms.Core/EasingTypeConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using static Xamarin.Forms.Easing;
+
+namespace Xamarin.Forms
+{
+	[Xaml.ProvideCompiled("Xamarin.Forms.Core.XamlC.EasingTypeConverter")]
+	[Xaml.TypeConversion(typeof(Easing))]
+	public class EasingTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (string.IsNullOrWhiteSpace(value))
+				return null;
+
+			value = value?.Trim() ?? "";
+			var parts = value.Split('.');
+			if (parts.Length == 2 && parts[0] == nameof(Easing))
+				value = parts[parts.Length - 1];
+
+			switch (value)
+			{
+				case string easing when AreEqual(easing, nameof(Linear)): return Linear;
+				case string easing when AreEqual(easing, nameof(SinIn)): return SinIn;
+				case string easing when AreEqual(easing, nameof(SinOut)): return SinOut;
+				case string easing when AreEqual(easing, nameof(SinInOut)): return SinInOut;
+				case string easing when AreEqual(easing, nameof(CubicIn)): return CubicIn;
+				case string easing when AreEqual(easing, nameof(CubicOut)): return CubicOut;
+				case string easing when AreEqual(easing, nameof(CubicInOut)): return CubicInOut;
+				case string easing when AreEqual(easing, nameof(BounceIn)): return BounceIn;
+				case string easing when AreEqual(easing, nameof(BounceOut)): return BounceOut;
+				case string easing when AreEqual(easing, nameof(SpringIn)): return SpringIn;
+				case string easing when AreEqual(easing, nameof(SpringOut)): return SpringOut;
+				default: throw new InvalidOperationException($"Cannot convert \"{value}\" into {typeof(Easing)}");
```

Comment:
I like that this doesn't use reflection, but it's not future proof, and could differ in behaviour with the compiled converter if/when someone add an easing and forget to add a case here.

what we do in `Color` e.g. is this kind of switch, plus a fallback using reflection as a safety net

Reply:
Hmm, how about writing the next test case.

With reflection, I will retrieve all public static read-only properties which have type "Easing".
Then we will pass their names into the original test-case.
And if an exception occurred, the test will be failed and it will be noticed (And no need for Reflection in Easing class).

I hope you got my idea

Reply:
you can't unfortunately test the fallback with an automatic unit test